scripts/join_structures_with_shapefile_areas.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+
+import geopandas as gpd
+import handling
+import polars as pl
+from fiona.drvsupport import supported_drivers
+from loguru import logger
+
+from ..nhs import config as user_config
+from ..nhs.data import geography
```

Comment:
This will fail since you can't use relative import for non python modules. import from `context` instead. `context.py` currently only imports `nhs` so you might need to import the other modules individually, see how `tests/` is set up (it's not the best way to write code, happy if you can suggest a better way :sweat: )

```python
from context import nhs

user_config = nhs.config
```